5-fibonacci/5-fibonacci/5-fibonacci.cpp

```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+#include <chrono>
+#include <vector>
+using namespace std;
+using namespace std::chrono;
+
+void IterateFibonacci(int a)
```

Comment:
This function does not return any value. Should it not return the result, just like the recursive function?